src/PointwiseFunctions/GeneralRelativity/Christoffel.hpp

```diff
@@ -38,4 +45,68 @@ template <size_t SpatialDim, typename Frame, IndexType Index, typename DataType>
 tnsr::abb<DataType, SpatialDim, Frame, Index> christoffel_first_kind(
     const tnsr::abb<DataType, SpatialDim, Frame, Index>& d_metric) noexcept;
 // @}
-} // namespace gr
+namespace Tags {
+/// Compute item for spatial Christoffel symbols of the first kind
+/// \f$\Gamma_{ijk}\f$ computed from the first derivative of the
+/// spatial metric.
+///
+/// Can be retrieved using `gr::Tags::SpatialChristoffelFirstKind`
+template <size_t SpatialDim, typename Frame, typename DataType>
+struct SpatialChristoffelFirstKindCompute
+    : SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>,
+      db::ComputeTag {
+  using argument_tags = tmpl::list<
+      ::Tags::deriv<gr::Tags::SpatialMetric<SpatialDim, Frame, DataType>,
+                    tmpl::size_t<SpatialDim>, Frame>>;
+  static constexpr tnsr::ijj<DataType, SpatialDim, Frame> (*function)(
+      const tnsr::ijj<DataType, SpatialDim, Frame>&) =
+      &christoffel_first_kind<SpatialDim, Frame, IndexType::Spatial, DataType>;
+  using base = SpatialChristoffelFirstKind<SpatialDim, Frame, DataType>;
```

Comment:
`base` and `type` are not part of the compute item interface. Please remove these.

Reply:
see [here](https://github.com/sxs-collaboration/spectre/pull/1478#discussion_r282530726)